charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -275,21 +275,21 @@ data:
 
     {{- end }}
 
-      # If we're in single namespace mode, we will also
-      # skip the configuration for elasticsearch. Houston will fallback
-      # to its default modes, which are false/disabled.
-      # XXX: We may want to decouple these configurations from the
-      # singleNamespace config. At the moment, this is a convenient
-      # helper to require a minimal config change to run the platform in
-      # single namespace mode. Fluentd requires a ClusterRole to
-      # enrich logs with kubernetes metadata. In a single namespace environment,
-      # we're likely not going to get a ClusterRole from admins.
-      {{- if not .Values.global.singleNamespace }}
-        elasticsearch:
-          enabled: true
-          connection:
-            host: {{ printf "%s-elasticsearch-nginx.%s" .Release.Name .Release.Namespace }}
-            port: 9200
+    # If we're in single namespace mode, we will also
+    # skip the configuration for elasticsearch. Houston will fallback
+    # to its default modes, which are false/disabled.
+    # XXX: We may want to decouple these configurations from the
+    # singleNamespace config. At the moment, this is a convenient
+    # helper to require a minimal config change to run the platform in
+    # single namespace mode. Fluentd requires a ClusterRole to
+    # enrich logs with kubernetes metadata. In a single namespace environment,
+    # we're likely not going to get a ClusterRole from admins.
+    {{- if not .Values.global.singleNamespace }}
+      elasticsearch:
```

Comment:
I'm having trouble working out the change here. Is this still under `airflow.elasticsearch`?

Reply:
so currently its not under airflow.elasticsearch but deployments.helm. this change slides it back to deployments

Reply:
the default.yaml file in houston is expecting it at the deployments level. And thus the default of host: localhost is not being over written